src/ts/uimanager.ts

```diff
@@ -461,6 +540,35 @@ export class UIManager {
   get onUiVariantResolve(): EventDispatcher<UIManager, UIConditionContext> {
     return this.events.onUiVariantResolve;
   }
+
+  /**
+   * Returns the list of all added markers in undefined order.
+   */
+  getTimelineMarkers(): TimelineMarker[] {
+    return this.config.metadata.markers;
+  }
+
+  /**
+   * Adds a marker to the timeline. Does not check for duplicates/overlaps at the `time`.
```

Comment:
Is there a use-case for not filtering exact duplicates before adding markers?

Reply:
Yes, I leave that responsibility to the user. If he really wants to have overlaps he can have them. But it does not make sense and will probably never happen in practice, because chapters or ad breaks seldomly overlap. That is unless somebody figures out that these markers can be used for some other creative stuff I didn't think of, but then we can always improve later.